jekyll/_cci2/fail-fast.adoc

```diff
@@ -0,0 +1,188 @@
+---
+version:
+- Cloud
+- Server
+---
+= Fail fast - Open preview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A new way to tighten your feedback loop when running tests on CircleCI.
+:icons: font
+:toc: macro
+:toc-title:
+
+
+WARNING: This feature is in **Open preview**, use at your own risk. This feature is not guaranteed to move to general availability. For questions and/or issues, please comment on our link:[Community forum] (add link).
+
+[#motivation-and-introduction]
+== Motivation and Introduction
+
+As a developer, speeding up your CI feedback loop is an excellent way to stay in your flow. For jobs with lengthy running tests, where even one failed test requires immediate attention, waiting for feedback until all tests have run to completion results in wasted time and unoptimized credit usage.
+
+Now in open preview is a new functionality to fail tests faster. You can configure your testing jobs to stop executing shortly after finding the first test failure.
```

Comment:
I am leaning towards rephrasing the first sentence. Something along the lines of... "CircleCI is releasing in open preview a new functionality to fail tests faster." I think the "now" makes it sound more like marketing copy.